README.md

```diff
@@ -52,7 +52,12 @@ func (p radiusService) RadiusHandle(request *radius.Packet) *radius.Packet {
 }
 
 func main() {
-	s := radius.NewServer(":1812", "sEcReT", radiusService{})
+	cls := new(radius.Clients)
+	cls.SetHerd([]radius.Client{
+		radius.NewClient("127.0.0.1", "secret1", "localhost"),
+		radius.NewClient("10.10.10.10", "secret2", "Some Client"),
+	})
+	s := radius.NewServer(":1812", "", radiusService{}, cls)
```

Comment:
The function of the secret and Clients is overlapping.
This will confuse the caller of radius.NewServer